docs/configuration/registration/json-dump.rst

```diff
@@ -0,0 +1,77 @@
+.. _configuration_registration_json_dump:
+
+=========
+JSON Dump
+=========
+
+The JSON Dump registration plugin can be used to send submitted form data as a
+`JSON object`_ to a configured service.
+
+What does the Open Forms administrator need?
+============================================
+Service options:
+
+* API root URL
+* API credentials (optional)
+* Client certificate (optional)
+* Server certificate (optional)
+
+Plugin options:
+
+* Which form variables to include in the data
+* API endpoint (optional)
+
+
+Configuration
+=============
+To configure the JSON Dump plugin, follow these steps:
+
+#. In Open Forms, navigate to: **Configuration** > **Services**
+#. Create a service to which the submitted form data should be sent.
+
+   a. Click **Add service**.
+   b. Fill out the form:
+
+      * **Label**: *Fill in a human readable label*, for example: ``JSON Dump service``
+      * **Type**: Select the type: ``ORC``
+      * **API root url**: The root of this API, *for example* ``https://example.com/objecten/api/v2/``
+      * **Authorization type**: Select the desired type of authorization
+
```

Comment:
Maybe we can add `* **Header key**: Fill in ``Authorization`` (optional)` or something like this in case we do have an authorization?